tembo-stacks/src/stacks/config_engines.rs

```diff
@@ -0,0 +1,513 @@
+use anyhow::Result;
```

Comment:
formerly tembo-operator/src/stacks/config_engines.rs

Reply:
Shouldn't it be a move/rename, then? Doing it this way will obliterate any history this file has.

Reply:
Depends.